infra/build/functions/build_project.py

```diff
@@ -166,6 +171,10 @@ def is_supported_configuration(build):
   fuzzing_engine_info = build_lib.ENGINE_INFO[build.fuzzing_engine]
   if build.architecture == 'i386' and build.sanitizer != 'address':
     return False
+  # TODO(jonathanmetzman): UBSan should be easy to support.
+  if build.architecture == 'aarch64' and (build.sanitizer
+                                          not in {'address', 'hwaddress'}):
```

Comment:
Do we want to just only support `hwaddress` here for arm?

Reply:
For now I'm not even building it, so let's punt this until later.